pkg/configmap/configmap_watcher.go

```diff
@@ -115,3 +161,12 @@ func (cmw *ConfigMapWatcher) getKubeLinterConfig(data string) (config.Config, er
 
 	return cfg, nil
 }
+
+func getPodNamespace() (string, error) {
+	namespace, exists := os.LookupEnv("POD_NAMESPACE")
```

Comment:
I think you forgot to add this new env variable to the deployment YAML definitions.